.github/workflows/build-action.yml

```diff
@@ -0,0 +1,84 @@
+name: Build Image
+
+# Trigger the workflow on push or pull request
```

Comment:
I do not like the idea to trigger on changes in any of our scripts. Too much traffic and useless image builds are caused by this - and it is still not complete, since any file in build-image folder may require a new image to be built.

I personally prefer manual builds of images.

Maybe a compromise could be to trigger on changes of build.bash. These are typically not that often, and any change which needs an image build could just add a newline to this file to trigger. WDYT?

Reply:
Not a bad idea, just make sure to remove extraneous newlines before merge.

Reply:
Should we make it rebuild on change to any files in the build image directory? Then we could add a dummy file to trigger builds.

Reply:
Holger's compromise sounds reasonable. Remember that while it's not *our* ressources, it's someone's.
And we have not had a real need to build another image in the last year(!).

No do not trigger on any change to build-image dir. I stopped counting how often I've touched first-boot.bash....
If you really want to force a build you can make a dummy change to build.bash.

Reply:
I am willing to live with that, does that seem acceptable @holgerfriedrich?